src/gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -196,6 +196,15 @@ properties:
     type: number
     minimum: 0
 
+  breslow_thickness_category:
+    $ref:
+      - "_terms.yaml#/breslow_thickness_category/common"
+    enum:
+        - <=1 mm
```

Comment:
@clsackett   This enum isn't surrounded by the single-quotes.  Why is that?  I always forget what the rules are here.

Reply:
Good to know!